src/components/Providers/index.tsx

```diff
@@ -23,9 +23,9 @@ function Providers({ children, store, styledTheme, muiTheme, history }: Provider
     <ThemeProvider theme={styledTheme}>
       <Provider store={store}>
         <MuiThemeProvider theme={muiTheme}>
-          <SnackbarProvider>
-            <Router history={history}>{children}</Router>
-          </SnackbarProvider>
+          <Router history={history}>
+            <SnackbarProvider>{children}</SnackbarProvider>
+          </Router>
```

Comment:
What was the issue with having it the other way around?

Reply:
You can't use `Link` components` outside of the `Router` provider.